tests/test_nifgen.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 from nitsm.pinquerycontexts import PinQueryContext
 
-OPTIONS = {"Simulate": True, "DriverSetup": {"Model": "5442", "BoardType": "PXIe"}}
+OPTIONS = {"Simulate": True, "DriverSetup": {"Model": "5433", "BoardType": "PXIe"}}

```

Comment:
this change is meant to hopefully fix intermittent failures with creating simulated nifgen sessions